src/triggers.js

```diff
@@ -722,6 +733,14 @@ function builtInTriggerValidator(options, request) {
       }
     }
   }
+  const userRoles = options.requireUserRoles;
+  if (userRoles) {
+    const roles = await auth.getUserRoles();
+    const hasRole = userRoles.some(requiredRole => roles.includes(`role:${requiredRole}`));
```

Comment:
Shouldn't require all of them?

Reply:
Maybe we could have:

requireUserRole: user just has to be one of many
requireUserRoles: user has to be all roles

Reply:
We've got similar functionality for our cloud functions and thought I could give some feedback (really excited about this feature as well!). What do you think about `requireAnyUserRoles` and `requireAllUserRoles`? My first thought was that the difference between them was only "single role" vs "many roles".

Also, we haven't really found a use case (for us) that requires a `requireAllUserRoles` api, but I guess every use case is different :)

Reply:
Thank you for your feedback @johanarnor! I like it. @davimacedo what are your thoughts?

Reply:
Yes. That looks a good idea.